draco/core/containers.py

```diff
@@ -1429,6 +1515,108 @@ def spectrum(self):
         return self.datasets["spectrum"]
 
 
+class DelayTransform(ContainerBase):
+    """Container for a delay transform of SiderealStream.
+    """
+
+    _axes = ("delay", "stack", "ra")
+
+    _dataset_spec = {
+        "vis": {
+            "axes": ["delay", "stack", "ra"],
+            "dtype": np.float64,
```

Comment:
Thanks for catching this one. `real_valued_output ` was not an input parameter for the first versions of `DelayTransformGibbs` and the output delay transform was forced to be real valued for sidereal stream inputs. Now that it is an input parameter it would fail if `real_valued_output=True`. I made the change so the `vis` dataset of `containers.DelayTransform` is complex valued.

Reply:
change pushed